test/e2e/nstemplatetier_test.go

```diff
@@ -30,11 +30,16 @@ const (
 
 func TestNSTemplateTiers(t *testing.T) {
 	// given
-	hostAwait, memberAwait, _ := WaitForDeployments(t)
+	hostAwait, memberAwait, memberAwait2 := WaitForDeployments(t)
 
 	// Create and approve "testingtiers" signups
 	testingTiersName := "testingtiers"
-	testingtiers := CreateAndApproveSignup(t, hostAwait, testingTiersName, memberAwait.ClusterName)
+	testingtiers, _ := NewSignupRequest(t, hostAwait, memberAwait, memberAwait2).
+		Username(testingTiersName).
+		ManuallyApprove().
```

Comment:
originally, the target cluster was set, so we should set it here too
```suggestion
		ManuallyApprove().
		TargetCluster(memberAwait).
```